### ISSUE TITLE
[FR] Several home directories

### ISSUE THREAD

**Issue by qinabu** [renerocksai/telekasten.nvim#96]
**Please confirm**
- [X] I am running the latest version of this plugin

**Is your feature request related to a problem? Please describe.**
It would be great to have the ability to switch between notebooks, for example, Personal and Work-related.

**Describe the solution you'd like**
It would be several tables for setup as an option and some selection in the Telescope.

**Comment by zhngjan** [comment]
I am looking for a note taking plugins that totally support markdown, then I found this one.
But I have more than one notebooks, so I want to switch my notebooks just like wiki home of vimwiki. It would be great to support that.

**Comment by lambtho12** [comment]
I think this was already discussed somewhere (in a PR or in discussions), but I can't find it.

**Comment by lambtho12** [comment]
Just thought about this.. I see multiple possibilities to solve this issue.

1. Add an option to the lua functions to tweak `cwd` at which the function is called.
That way the config could be kept clean with only one 'main' home, but you could define other mappings with custom parameters such as:
```vim
nnoremap <leader>zf :lua require('telekasten').find_notes()<CR>
nnoremap <leader>zfW :lua require('telekasten').find_notes({cwd = '~/path/to/work/dir'})<CR>
nnoremap <leader>zfS :lua require('telekasten').find_notes({cwd = '~/path/to/shared/dir'})<CR>
```
The benefit of option 1 is that we can work seamlessly with multiple homes at the same time (just call the correct mapping). The drawback is that it requires the addition of new mappings, which could be an issue if people have a large number of home directories (one per client or one per project for instance).

2. Add a function to overwrite the `M.Cfg.home` value and have all pickers work with the new home instead (that could be a problem for the journal dir and the checks if dir exist). 
What would be nice with that is that we could have the user write multiple home paths in a table and use a telescope picker to select the one we want. With that we would have just a new function and one or two mappings (one to run the picker and select the new home, one to get back to the default home).

The benefit of option 2 is that we just need to add one or two mappings and it can accommodate a very large number of homes without issues. But then it becomes tedious to work between two different homes at the same time.



I do not really have a use for it, so I may be overseeing some critical stuff here. What do you think? In any case, both options seem quite simple to implement and should not break any features for people using just one home.

**Comment by lambtho12** [comment]
PR #158 proposes a solution for this.

**Comment by lambtho12** [comment]
Closed with commit fa667469815e3cf6d25a7b09e7e4ce31036c197c (squashed from PR #158)
